### ISSUE TITLE
`jss setup` does not work with Sitecore Docker

### ISSUE THREAD

**Issue by peternazarov** [Sitecore/jss#341]
## Description
Docker deployment directory does not suppose to contain web.config or any other standard Sitecore files. Reference: https://github.com/Sitecore/docker-images

## Expected behavior
Docker deployment directory should be accepted without making a check for the default Sitecore files in that directory.

## Steps To Reproduce
run `jss setup` and use Sitecore Docker deployment directory.

## Possible Fix
Remove Sitecore files check in the deployment directory.

## Your Environment
latest as of today. others are N/A.

## Screenshots
![image](https://user-images.githubusercontent.com/21163169/76343685-c2f1d580-6300-11ea-857c-c6e060656c33.png)
 ```
jss setup
Is your Sitecore instance on this machine or accessible via network share? [y/n]: y
Path to the Sitecore folder (e.g. c:\inetpub\wwwroot\my.siteco.re): C:\Projects\SitecoreHelixSolutionTemplateTest2\docker\deploy
C:\Projects\SitecoreHelixSolutionTemplateTest2\docker\deploy did not look like a valid Sitecore website or headless proxy (missing 'sitecore' folder, Web.config, or package.json)!```

**Comment by sshushliapin** [comment]
Faced the same issue. Creating empty `sitecore` directory did the trick (as mentioned in the error message).

But, I do agree with you that this check looks awkward in case of Docker environment. My vote is for getting rid of it.

**Comment by sc-illiakovalenko** [comment]
Fixed in #361
